packages/components/organisms/f-footer/src/components/Footer.vue

```diff
@@ -222,6 +223,12 @@ $footer-heading-font-size: 'heading-s';
     }
 }
 
+.c-footer-row-item--fullWidthOnDesktopView {
```

Comment:
Maybe `c-footer-row-item--fullWidth--aboveWide` or `c-footer-row-item--fullWidthAboveWide`?

Reply:
Agree – try not to use device's as part of classnames and go for using breakpoint names

Reply:
:+1: Changed